### PR TITLE
Use new recovery function in client module

### PR DESCRIPTION
**Pull request by blakecduncan**
- Using the new recovery function in the client module
- Increasing the rpc polling interval so we don't spam our RPC

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -39,3 +39,5 @@ export function getNetwork(networkName: string) {
 }
 
 export const BLS_TEAM_PK = process.env.REACT_APP_BLS_TEAM_PK ?? '';
+
+export const RPC_POLLING_INTERVAL = 10000;
```
Comment (blakecduncan) [inline src/constants.ts:43]: I think by default this is polling every second which is why we see such large numbers.  This now is polling every 10 seconds

**Comment by jacque006** [comment]
@blakecduncan Are we going to do a similar update in the extension as well?

**Comment by blakecduncan** [comment]
> 

@jacque006  Yeah, Quill is already using the new recovery methods.  I just figured out the RPC thing yesterday when I was playing with the instant wallet.  I'll make this update in Quill next

**Review by JohnGuilding — APPROVED** [review]
LGTM

File: src/constants.ts (hunk 1 above)
Comment (voltrevo) [reply src/constants.ts:43]: I think 10s is a bit slow :(. ethers default should be 4s which is already a bit of a drag. When polling it's just asking whether there's a new block, and only follows up if there is.

Comment (blakecduncan) [reply src/constants.ts:43]: @voltrevo Yeah, I didn't put a lot of thought into what number to pick.  I don't think there's really anyone, other than a few of us for testing, using this wallet so a just picked an arbitrarily high number.

I didn't realize the default was at 4s till I did a deeper dive in Quill.  Maybe it's not worth updating the polling interval

**Pull request merged**